javasdk-async/src/main/java/io/vantiq/svcconnector/RequestProcessingVerticle.java

```diff
@@ -0,0 +1,81 @@
+package io.vantiq.svcconnector;
+
+import io.reactivex.rxjava3.core.Completable;
+import io.vertx.core.AbstractVerticle;
+import io.vertx.core.Context;
+import io.vertx.core.Promise;
+import io.vertx.core.Vertx;
+import io.vertx.ext.web.Router;
+import io.vertx.ext.web.handler.sockjs.SockJSHandler;
+import io.vertx.ext.web.handler.sockjs.SockJSHandlerOptions;
+import io.vertx.ext.web.handler.sockjs.SockJSSocket;
+import io.vertx.ext.web.sstore.LocalSessionStore;
+import io.vertx.ext.web.sstore.SessionStore;
+import lombok.Getter;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public class RequestProcessingVerticle extends AbstractVerticle implements Sessionizer {
```

Comment:
Class comment (I'll refrain from repeating this, but probably would be good for all classes of any size).